config/karma.saucelabs.js

```diff
@@ -119,19 +114,50 @@ module.exports = function(config) {
 
     customLaunchers: sauceLabsBrowsers,
 
-    reporters: ['spec', 'saucelabs'],
+    reporters: ['spec', 'summary', 'saucelabs'],
 
     port: 9876,
 
-    retryLimit: 3,
+    retryLimit: 0,
 
     // concurrency: 10,
 
+    client: {
+      mocha: {
+        timeout: 20000,
+        retries: 3,
+        grep:
+          'Crawler Support|local timestamp|raises error event|handles failures',
+        invert: true
+      }
+    },
+
+    browserConsoleLogOptions: { terminal: false },
+
+    specReporter: {
+      maxLogLines: 5,
+      suppressErrorSummary: true,
```

Comment:
Out of curiosity, why do we want to suppress the error summary? That sounds useful offhand.

Reply:
We actually have another reporter below that provides a better summary of the run, as which test failed in which browser. So the summary from this reporter is not of much use. But there's no harm to turn it on.